packages/html/src/pager/pager.spec.tsx

```diff
@@ -36,8 +37,9 @@ export type KendoPagerState = { [K in (typeof states)[number]]?: boolean };
 
 const defaultOptions = {
     size: Size.medium,
-    adaptive: false,
-    showPagerSizeInfo: true,
+    pagerSizeInfo: true,
```

Comment:
do we need the "Info" here? Wouldn't it be clearer if we leave these as "pagerSize, pageTitle, maxPages"?